index.html

```diff
@@ -2355,13 +2355,23 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the sender of a pre-signed
+message to reveal parts of the message without revealing the entire message. For
+example, selectively disclosing a digital driver's license for the purposes of
+renting a car might involve the entity presenting the license to only reveal the
+issuing authority, license number, birthday, and authorized motor vehicle class.
+Note that in this case, the license number is correlatable information that is
+being shared but some amount of privacy is still being preserved because the
+driver's full name and address are not being shared.
+        </p>
+
+        <p>
+Selective disclosure is a technique that not all cryptosuites provide. It is
+also a technique that the creator of the initial message needs to utilize to
+ensure that entities that re-transmit the message have the functionality
+available to them. A cryptosuite that provides support for selective
+disclosure is `bbs-2022`.
```

Comment:
```suggestion
Not all cryptosuites are capable of providing selective disclosure. The
author of a message needs to enable selective disclosure on it in order
for a presenter of the message to use the technique. One cryptosuite
that supports selective disclosure is `bbs-2022`.
```

Reply:
starting from @dlongley's suggestion --
```suggestion
Not all cryptosuites are capable of providing selective disclosure. One
cryptosuite that does support selective disclosure is `bbs-2022`.
        </p>
        <p>
The author of a message needs to use a capable cryptosuite and enable
selective disclosure on the message in order for a recipient of that
message to use the technique on future presentations.
```